### ISSUE TITLE
words are sometimes duplicated when using windows speech to text functionality

### ISSUE THREAD

**Issue by RyanCavanaugh** [microsoft/vscode#181664]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.78.0
- OS Version: Windows 11

Steps to Reproduce:

1. New markdown file
2. Press Windows-H to open speech recognition
3. Start saying some words with varying pacing. Depending on timing, you will see some words appear twice in the buffer even though they were only spoken once. This is specific to Visual Studio Code and does not happen in other applications such as notepad.

I've also had it seemingly just entirely corrupt the buffer, leaving it in what appears to be an intermediate state where a line disappears when the carrot is on it but otherwise appears with the speech ellipses still present

**Comment by RyanCavanaugh** [comment]
here it is duplicating an entire line of text, for example

https://user-images.githubusercontent.com/6685088/236563532-dba4c96a-7091-45ce-aca3-aa6fd832f17c.mp4

**Comment by isidorn** [comment]
@RyanCavanaugh thanks for reporting this. Was this working previously? I wonder if it is an Electron 22 update regression.

Assigning to June so we reserve time next milestone to investigate deeper.

**Comment by RyanCavanaugh** [comment]
I don't know, I haven't used the feature until this week

**Comment by meganrogge** [comment]
@RyanCavanaugh can you pls try to repro this on prior versions? thanks

**Comment by RyanCavanaugh** [comment]
Do you have a particular version in mind, and instructions on how to install it?

**Comment by RyanCavanaugh** [comment]
@meganrogge this seems to only list a single (latest) version, but you asked me to try on prior versions. Can you clarify?

**Comment by meganrogge** [comment]
@RyanCavanaugh my bad 😅 and thanks for following up. You can find older versions here for example

https://code.visualstudio.com/updates/v1_77

**Comment by meganrogge** [comment]
![Screenshot 2023-06-12 at 1 56 28 PM](https://github.com/microsoft/vscode/assets/29464607/259cac7a-af2e-4577-bcd6-f31f85eeed57)

**Comment by RyanCavanaugh** [comment]
1.77
```
This is me testing out the newest feature 
This is me testing out the newest feature 
This is me testing out the newest feature 
 
```

**Comment by RyanCavanaugh** [comment]
Why are you calling it a regression when it's never worked in any version of VS Code? 😕

**Comment by meganrogge** [comment]
Oh thanks for clarifying. I was under the impression that the behavior in your earlier comment was expected in the June 2021 version.

**Comment by meganrogge** [comment]
I am still not sure why this wouldn't be working in VS Code specifically if it does in other apps. My best guess is something with electron

**Comment by deepak1556** [comment]
I am unable to repro the issue with latest stable `1.85`, @RyanCavanaugh can you confirm. Thanks!

**Comment by RyanCavanaugh** [comment]
Yep, seems fine in a brief test
